spec/Process/ProcessBuilderSpec.php

```diff
@@ -27,9 +27,19 @@ function it_is_initializable()
 
     function it_should_be_able_to_create_process_arguments_based_on_taskname(ExternalCommand $externalCommandLocator)
     {
-        $externalCommandLocator->locate('grumphp')->willReturn('/usr/bin/grumphp');
+        $externalCommandLocator->locate('grumphp', false)->willReturn('/usr/bin/grumphp');
 
-        $arguments = $this->createArgumentsForCommand('grumphp');
+        $arguments = $this->createArgumentsForCommand('grumphp', false);
+        $arguments->shouldHaveType(ProcessArgumentsCollection::class);
+        $arguments[0]->shouldBe('/usr/bin/grumphp');
```

Comment:
`$arguments->shouldHaveKeyWithValue(0, '/usr/bin/grumphp');`

Reply:
To me, it's more readable as it is right now

Reply:
👍